insights/parsers/nmcli.py

```diff
@@ -176,10 +176,20 @@ class NmcliConnShow(CommandParser):
         data (list): list of connections wrapped in dictionaries
 
     """
+
     def parse_content(self, content):
-        try:
-            self.data = parse_delimited_table(content, heading_ignore=["NAME", "UUID", "TYPE", "DEVICE"])
-        except:
+        self.data = []
```

Comment:
@TZ3070 - better fix this issue #3461 in an individual PR but not mix them together.